suites/reef/cephfs/tier-2_cephfs_journal_tool.yaml

```diff
@@ -0,0 +1,119 @@
+---
```

Comment:
why do we need a new suite file ?

tier-4_cephfs_recovery.yaml - can we reuse this suite file?